dev-requirements.txt

```
@@ -1,7 +1,7 @@
 asgiref==3.5.0
 attrs==21.4.0
 coverage==6.3.2
-Django==3.2.18
+Django==3.2.19
```

```suggestion
Django>=3.2.,<4.0
```
let's fix this for good